hazelcast/src/main/java/com/hazelcast/map/impl/MapServiceContextImpl.java

```diff
@@ -329,15 +330,16 @@ public void flushMaps() {
 
     @Override
     public void destroyMap(String mapName) {
+        // on LiteMembers we don't have a MapContainer, but we may have a Near Cache and listeners
+        mapNearCacheManager.destroyNearCache(mapName);
+        nodeEngine.getEventService().deregisterAllListeners(SERVICE_NAME, mapName);
```

Comment:
I'm not sure about this change though. Does this remove listeners this LiteMember has on other members? Or does this remove listeners which other nodes or clients have on this LiteMember? Actually all tests pass regardless where this line is, so we might miss another test for what this line actually does.